src/yang/ietf-tcp.yang

```diff
@@ -270,6 +270,58 @@ module ietf-tcp {
     }
   } // grouping tcp-ecn-grouping
 
+  // congestion control
+
+  identity congestion-control-algorithm {
+    description
+      "Base identity from which all congestion control algorithms
+       are derived.";
+  }
+
+  identity Reno {
+    base congestion-control-algorithm;
+    description
+      "Standard TCP congestion control (RFC 5681) referred to as
+       'Reno algorithm'.";
+  }
+
+  identity NewReno {
+    base congestion-control-algorithm;
+    description
+      "NewReno modification to TCP's fast recovery algorithm
+       (RFC 6582)'.";
```

Comment:
Instead of citing RFCs within a description statement, would suggest add a reference statement.

reference 
  "RFC 6582: The NewReno Modification to TCP.";